packages/components/tokens/components/button.js

```diff
@@ -7,11 +7,11 @@ module.exports = {
     paddingRight: { value: `${elements.space.plus2}` },
     fontSize: { value: `${elements.typography.fontSizes.base}` },
     minHeight: { value: `${elements.size.plus4}` },
-    background: { value: `${elements.color.white}` },
-    color: { value: `${elements.color.primary}` },
+    background: { value: `${elements.color.white.base}` },
+    color: { value: `${elements.color.primary.base}` },
     borderWidth: { value: `1px` },
     borderStyle: { value: `solid` },
-    borderColor: { value: `${elements.color.primary}` },
+    borderColor: { value: `${elements.color.primary.base}` },
```

Comment:
There should be no `.base` color, we decided to use `.main`

Reply:
Fixed!